### PR TITLE
feat(merge-guard): Component 2b clean-pass/trigger-comment exemptions

### PR DESCRIPTION
**Pull request by scotthamilton77**
## Summary
- `untriaged_feedback` blocker now exempts two shapes of issue comment the Codex re-review loop itself generates: (1) the clean-pass announcement (allowlisted bot identity + pinned marker prefix), (2) the `@codex review` trigger comment (PR author or the authenticated session identity). Without this, asking for review or getting a clean pass trips the blocker it exists to satisfy.
- Two hardening tests added for the already-merged (.1) reaction-path predicate: a `gh api` failure fixture (exit 3, fail-closed), and a `+1 created_at == last_head_change` boundary case (must reject — strict `>` only).
- One-line spec residual note: a non-force push with a backdated `GIT_COMMITTER_DATE` is out of the cooperative-loop threat model — accepted explicitly.

## Split out
The review_summary ratchet (bead .2's original part (c) — each Codex round mints a new review_summary id, so a triage inventory from round N never covers round N+1) is split into sibling bead `agents-config-abn9.44.2.8`. It roughly doubles this diff and needs a new fact (a genuinely-clean signal distinct from `bot_clean_review_at_head`, which only means "APPROVED/COMMENTED", not "no findings") plus round-ordering tests — a design cycle, not a fold-in.

## Design
`docs/specs/2026-07-18-codex-rereview-path-design.md`, Component 2b.

## Test plan
- [x] `check-merge-eligibility_test.sh`: 205/205 assertions pass, 0 failures, exit 0.
- [x] Verified the `AUTH_LOGIN` lookup-failure degradation is fail-closed for the merge gate: on failure it becomes `""`, which can never match a real GitHub login, so the trigger-comment exemption simply does not fire and the comment stays counted as live/untriaged (blocks, never wrongly unblocks).

bead: agents-config-abn9.44.2.2

### CODE REVIEW

**Comment by scotthamilton77** [comment]
@codex review

File: src/user/.agents/skills/merge-guard/check-merge-eligibility.sh [hunk 1]
```diff
@@ -614,7 +628,33 @@ done < <(find "${HOME}/.claude/state/pr-inventory" -maxdepth 1 \
...
+            and ((.body // "") | startswith("Codex Review: Didn'\''t find any major issues"));
+        # (2) Trigger comment: the literal "@codex review" ask this repo'\''s
+        # own lieutenant workflow posts via `gh pr comment` to request a
+        # re-review — authored by either the PR author (a human re-asking)
+        # or the currently authenticated session identity (an agent
+        # re-asking as the CLI user). Matched on the same prefix
+        # request-rereview.sh posts verbatim.
+        def is_trigger_comment:
+            ((.user.login == $pr_author) or (.user.login == $auth_login))
+            and ((.body // "") | startswith("@codex review"));
```
[the remark this answers is not in the document]
Comment (scotthamilton77) [reply src/user/.agents/skills/merge-guard/check-merge-eligibility.sh]: Fixed in 3d22e4a — the trigger-comment exemption now requires an exact (whitespace-trimmed) match against `@codex review`, not a prefix match. request-rereview.sh always posts that exact body with no trailing text, so a comment like '@codex review — also fix the race below' no longer qualifies and stays counted as untriaged feedback. New test: 'trigger phrase plus appended feedback still blocks (exact match only)'. 206/206 assertions pass.

**Comment by scotthamilton77** [comment]
@codex review
